core/chains/terra/terratxm/txm.go

```diff
@@ -83,7 +82,7 @@ func NewTxm(db *sqlx.DB, tc terraclient.ReaderWriter, fallbackGasPrice string, g
 		confirmPollPeriod:  1 * time.Second,
 		confirmMaxPolls:    100,
 		fallbackGasPrice:   sdk.NewDecCoinFromDec("uluna", fgp),
-		gasLimitMultiplier: gasLimitMultiplier,
+		gasLimitMultiplier: cfg.GasLimitMultiplier,
```

Comment:
I stopped short of refactoring the txm here, but we may want to pass along a config interface so that things can be reconfigured live, without restarting the node.

Reply:
I'm thinking we should put confirmPollPeriod and confirmMaxPolls in the config as well.

Reply:
Added